src/main/java/me/lucien/minesweeper/domain/Room.java

```diff
@@ -154,4 +151,24 @@ public void flag(int x, int y) {
 
         return res;
     }
+
+    public int getRoomId() {
```

Comment:
We'd better avoid wordy method name. For example, `room.getRoomId()`.
There will be two "room" in the statement. Ideally `getId()` will be the best, unless there are other ids.

Reply:
Room class already has a static auto-incrementing id.

Reply:
Rename it to something else. It's a private static id anyway.